aiohttp_sse/__init__.py

```diff
@@ -142,8 +142,16 @@ async def wait(self) -> None:
         """
         if self._ping_task is None:
             raise RuntimeError("Response is not started")
-        with suppress(asyncio.CancelledError):
+
+        try:
             await self._ping_task
+        except asyncio.CancelledError:
+            if (
+                sys.version_info >= (3, 11)  # TODO(PY311): remove this condition
+                and (task := asyncio.current_task())
+                and task.cancelling()
```

Comment:
The version_info suffices as a TODO.
```suggestion
            if (
                sys.version_info >= (3, 11)
                and asyncio.current_task().cancelling()
```

Reply:
mypy said that `asyncio.current_task()` returns `Optional[Task]`, so I can't call `None.cancelling()`

Reply:
Ah, right. Technically true from a typing perspective, but impossible to happen here...
I guess that's fine.

Reply:
```suggestion
            if (
                sys.version_info >= (3, 11)
                and (task := asyncio.current_task())
                and task.cancelling()
```

Reply:
Sure, but it's more clear than add # type: ignore :D